src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
Ok. Shall I remove the option then to have a systemwide availability in /var/lib/dosemu/inst/dos or do you think that should be kept in another way as well? Having both is slightly non-trivial.